### PR TITLE
Sellers can add and view sub-products

### PR DESCRIPTION
**Pull request by ethanschreur**
**What does this PR do?**
- This pull request allows sellers to add and view subproducts

**Description of the task to be completed** 
- Fill out and submit an add subproducts form
- View the added subproduct data in the product visualization

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/login
- Enter the correct email and password and submit them
- View the dashboard page
- Press the products button
- Fill out the "add products" form and submit it
- Click on the added product in the products table
- Fill out the "add subproducts" form and submit it
- View the added subproduct information in the product visualization section

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -44,14 +44,12 @@ def products():
     if ("seller_email" not in session):
         return redirect('/login')
     products = Product.query.all()
-    print(products)
```
Comment (alexkayabula) [inline app/__init__.py:47]: @ethanschreur How did we miss this !!!!

File: app/__init__.py [hunk 2]
```diff
@@ -44,14 +44,12 @@ def products():
...
     return render_template('seller/products.html', products=products)
 
 @app.route('/products', methods=['POST'])
 def add_product():
     if ("seller_email" not in session):
         return redirect('/login')
-    print(request.form)
```
Comment (alexkayabula) [inline app/__init__.py:54]: @ethanschreur How did we miss this !!!!

**Review by alexkayabula — APPROVED** [review]
LGTM. Thank you.

File: app/__init__.py (hunk 2 above)
Comment (ethanschreur) [reply app/__init__.py:54]: Oops. I'll fix that for the next PR.

**Comment by alexkayabula** [comment]
It is ok. I merged the change.

**Pull request merged**